### ISSUE TITLE
[DateTimeFormat bug] Weekday short translation for 'pt-PT' returns the long translation

### ISSUE THREAD

**Issue by sizcovich1** [mdn/translated-content#23175]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

### What specific section or headline is this issue about?

This is about the Intl.DateTimeFormat library

### What information was incorrect, unhelpful, or incomplete?

When we do:

```
var currentDate = new Date()
console.log(new Intl.DateTimeFormat('pt-PT', { weekday: 'short' }).format(currentDate.getDate() + 1));
```
we get `quarta` 

If we replace the locale with `pt-BR`, we get `qua`. We need the `pt-PT` weekdays short format to work.

### What did you expect to see?

The short version of the weekdays in `pt-PT`

**Comment by yin1999** [comment]
Hey @sizcovich1. I believe this issue is related to MDN. We only document Web features and Web-related tech. You may want to report this with [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html) or other browser which does not have the correct behavior.

I'm going to close this. Feel free to ask for help if you got any other problem :)
